src/Modules/Backend/assets/Backend/webpack/js/Components/ToggleSecret.js

```diff
@@ -0,0 +1,23 @@
+export default class ToggleSecret {
```

Comment:
TogglePasswordInputType seems like a better class name that is more descriptive

Reply:
And maybe also add this to the core components since I can see a use for this as well in the frontend if needed

Reply:
Fixed in [8f4137e](https://github.com/forkcms/forkcms/pull/3560/commits/8f4137eac1b25aec5dea5c703c88afa53c0b71b5) and [e8bab89](https://github.com/forkcms/forkcms/pull/3560/commits/e8bab89fee74ab2f9f37641b21d12be2068b6a87). The only thing now is that the "trigger" JS is duplicated in Frontend.js and Backend.js. Plus the block template for the form type is duplicated in (Frontend)FormLayout and (Backend)formTheme.